core/src/com/po/fuck/model/collections/All.java

```diff
@@ -5,4 +5,8 @@ public final class All {
     public static DrawableCollection drawableCollection = new DrawableCollection();
     public static UpdatableCollection updatableCollection = new UpdatableCollection();
     public static EntityCollection entityCollection = new EntityCollection();
```

Comment:
Add `All` to Core. Make these fields non-static?

Reply:
Make it static inside of the `Core` ?

Reply:
Make it non-static inside of `Core`...

Reply:
It is accessed from a lot of places, for example from a `BasicEnemy`